tools/bridge/bridge/confirmation_task_planner.py

```diff
@@ -14,31 +14,28 @@ class ConfirmationTaskPlanner:
     def __init__(
         self,
         sync_persistence_time: float,
+        minimum_balance: int,
+        control_queue: Queue,
         transfer_event_queue: Queue,
         home_bridge_event_queue: Queue,
         confirmation_task_queue: Queue,
     ) -> None:
-        self.recorder = TransferRecorder()
+        self.recorder = TransferRecorder(minimum_balance)
```

Comment:
I think we might eventually better pass a TransferRecorder instance instead of adapting the signature every time the transfer recorder needs a new argument.But for the moment we can also keep it like it is.